com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/Fog.cs

```diff
@@ -200,4 +218,71 @@ public sealed class FogColorParameter : VolumeParameter<FogColorMode>
         public FogColorParameter(FogColorMode value, bool overrideState = false)
             : base(value, overrideState) { }
     }
+
+    /// <summary>
+    /// This enum defines the two modes for controlling the quality and cost of the volumetric fog.
+    /// </summary>
+    public enum FogControl
+    {
+        /// <summary>
+        /// In this mode, the user changes the parameters on higher abstraction level centered around performance.
```

Comment:
Use this mode if you want to change the fog control properties based on a higher abstraction level centered around performance.